### PR TITLE
add nats cluster

### PR DESCRIPTION
**Pull request by andriisoldatenko**
The purpose of this PR is just to demonstrate how to run nats cluster with nats-streaming cluster locally

### CODE REVIEW

File: docker-compose-nats-cluster.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    image: postgres
+    restart: always
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+    ports:
+    - 5432:5432
+    volumes:
+      - postgres:/var/lib/postgresql/data
+  nats:
```
Comment (adam2k) [inline docker-compose-nats-cluster.yml:29]: Could we do one of the two options:

1. Add a comment above `nats` or at the top of this file that explains when to use this over the basic `docker-compose`?
2. If it doesn't impact our systems much, I'd recommend that we convert the current `docker-compose.yml` to this version so we will have one less local dev env option.

File: docker-compose-nats-cluster.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
+version: "3"
```
Comment (adam2k) [inline docker-compose-nats-cluster.yml:1]: This is what I mean for suggestion #1 in the comment below.

```Suggestion
# This file is intended to use for debugging NATS Streaming (STAN) issues.  Use docker-compose.yml for local development.
version: "3"
```

Comment (andriisoldatenko) [reply docker-compose-nats-cluster.yml:2]: @adam2k pls check my version.

**Comment by schnie** [comment]
I guess this drops code coverage? Looks like that's why its failing. Can we ignore this file or something?

**Comment by andriisoldatenko** [comment]
@schnie PTAL. fixed codecoverage.

**Pull request merged**